roles/pulibrary.approvals/meta/main.yml

```diff
@@ -16,4 +16,21 @@ dependencies:
   - role: 'pulibrary.rails-app'
   - role: 'pulibrary.mailcatcher'
   - role: 'pulibrary.postgresql'
-  - role: 'pulibrary.samba'
\ No newline at end of file
+    vars:
```

Comment:
@kayiwa why not put these in ` roles/pulibrary.approvals/vars/default.yml`?  I would never look for vars here.  Is this a new best practice?

Reply:
It isn't baked yet but yes @carolyncole I can't recall anymore but I think I tried it failed.